woocommerce-addressfinder.php

```diff
@@ -68,13 +68,25 @@ function add_addressfinder_widget( $_checkout ) {
 		wp_enqueue_script( 'addressfinder_js', plugins_url( 'addressfinder.js', __FILE__ ), array(), ADDRESSFINDER_WOOCOMMERCE_VERSION, true );
 	}
 
-	add_filter( 'woocommerce_get_settings_checkout', 'addressfinder_settings', 10, 1 );
+	// Add the tab to the tabs array
+	function filter_addressfinder_settings_tabs_array( $settings_tabs ) {
+    $settings_tabs['addressfinder-settings'] = __( 'Addressfinder Settings', 'woocommerce' );
+
+    return $settings_tabs;
+	}
+	add_filter( 'woocommerce_settings_tabs_array', 'filter_addressfinder_settings_tabs_array', 99 );
```

Comment:
Whats `99`? Priority?

Reply:
Order if I got that right. So that'll be the last tab item